lib/datadog/tracing/contrib/aws/configuration/settings.rb

```diff
@@ -32,6 +32,8 @@ class Settings < Contrib::Configuration::Settings
                 )
               end
             end
+
+            option :peer_service
```

Comment:
I think we should add an ENV var to back this value, to make it configurable without code. e.g. `DD_TRACE_AWS_PEER_SERVICE`. A code comment (explaining usage, effect, purpose) and addition to the AWS options table in `GettingStarted.md` would also be in order.

Same goes for the rest.

Reply:
Hmm ok if that's confirmed what we want to do, we would need to utilize the new options which is possible but just a thing to note. Then we would probably create a separate test that tests the env var. Will do though, saw this comment after I pushed my other changes

Reply:
Is there a reason why would wouldn't want to do this?

Reply:
Nope, sounds like a good idea to me. I implemented it thoroughly, but we need to discuss our approach on how to document this properly in terms of comments and documentation